src/test.ts

```diff
@@ -34,10 +34,13 @@ export async function test(fn_name: string, test_req: any): Promise<string> {
       headers,
     });
     const response = await request;
-    console.log('request:', request);
     if (typeof request.text !== 'function') {
       return `Encountered an ${response.statusText} trying to test ${fn_name}}`;
     }
+
+    if (response.status !== 200) {
+      return `function returned ${response.status} error.`;
```

Comment:
Do we know _which_ function? Would be better to include: 

"Function 'fn-prevalence-predictor' returned ... error"

Unless the 'fn-prevalence-predictor' part is already visible on the page when the error would show up?